h/static/scripts/test/time-test.js

```diff
@@ -50,6 +51,34 @@ describe('time', function () {
   });
 
   describe('.toFuzzyString', function () {
+
+    var originalIntl;
+
+    before(function () {
+
+      // Intl isn't implemented in PhantomJS so we have to mock it in these
+      // tests. The mock version returns the results we'd expect from the real
+      // one for the test values that we use.
+      originalIntl = window.Intl;
```

Comment:
Rather than stubbing `window.Intl` here you could make it an optional third argument to the `format()` function which defaults to `window.Intl` if undefined. eg:

```js
function format(date, options, Intl) {
  if (typeof Intl === 'undefined') {
    Intl = window.Intl;
  }
  if (Intl && Intl.DateTimeFormat) {
   ...
  } else {
   ...
  }
}
```

Then you can test the two cases via `format(date, options, FakeIntl)` and `format(date, options, null)`